features/order/managing_orders/cancelling_order.feature

```diff
@@ -28,3 +28,10 @@ Feature: Cancelling orders
         Given the customer canceled this order
         When I view the summary of the order "#00000022"
         Then I should not be able to cancel this order
+
+    @ui
+    Scenario: Checking order payment state of a cancelled order
+        Given I view the summary of the order "#00000022"
```

Comment:
Do we need it? There's "When I browse orders" step below. If it's needed for "I cancel this order" then we should consider making it a one step "the order #42 was cancelled".